include/FastBVH/BVH.h

```diff
@@ -20,10 +20,15 @@ template <typename Float>
 struct BVHFlatNode final {
   //! The bounding box of the node.
   BBox<Float> bbox;
+
   //! The index of the first primitive.
   uint32_t start;
+
   //! The number of primitives in this node.
   uint32_t nPrims;
+
+  //! Number of elements to skip in flattened tree to get to a left child's sibling. 
+  //! (Node+1 == Node's left child , Node + rightOffset == Node's right child)
```

Comment:
This is extremely helpful. I had misunderstood this before and now it's very clear. Thanks!